lmnet/lmnet/networks/object_detection/yolo_v1.py

```diff
@@ -98,6 +98,7 @@ def metrics(self, output, labels, thresholds=[0.3, 0.5, 0.7]):
         for overlap_thresh in thresholds:
             average_precisions = []
             for class_id, class_name in enumerate(self.classes):
+                class_name = class_name.replace(' ', '_')
```

Comment:
@Joeper214 
Thank you for fixing!!

One question.
In you code, same changes exist in two networks.
It's not possible to fix in one change? (maybe in base class?)

Reply:
Yeah. That's a better idea. I was thinking about using Lambda function on base class. to filter or replace spaces with an underscore.

Reply:
@iizukak Updated in the next commit thanks!